geode-core/src/test/java/org/apache/geode/test/dunit/cache/internal/JUnit4CacheTestCase.java

```diff
@@ -186,20 +167,18 @@ public static final synchronized void beginCacheXml() {
    */
   public final void finishCacheXml(final String name) {
     synchronized (JUnit4CacheTestCase.class) {
-      File file = new File(name + "-cache.xml");
       try {
+        File file = new File(name + "-cache.xml");
         PrintWriter pw = new PrintWriter(new FileWriter(file), true);
         CacheXmlGenerator.generate(cache, pw);
         pw.close();
-      } catch (IOException ex) {
-        // TODO: remove error handling
-        Assert.fail("IOException during cache.xml generation to " + file, ex);
-      }
-      // TODO: System.setProperty(GEMFIRE_PREFIX + CACHE_XML_FILE, file.getAbsolutePath());
-      cache = null;
-      GemFireCacheImpl.testCacheXml = file;
-      try {
+        // TODO: System.setProperty(GEMFIRE_PREFIX + CACHE_XML_FILE, file.getAbsolutePath());
```

Comment:
@jaredjstewart do you know what this is about?

Reply:
I don't, none of the M&M tests I have ever seen extend this class.  I'd say delete anything that's commented out.